Documentation/user-guides/getting-started.md

```diff
@@ -146,7 +146,7 @@ spec:
         app: example-app
     spec:
       containers:
-      - name: example-app 
+      - name: example-app
```

Comment:
This is an embedded yaml located in `../../example/user-guides/getting-started/example-app-deployment.yaml`, could you make this whitespace change there as well? Then CI should be green.